controller/handler/trial.js

```diff
@@ -124,6 +127,10 @@ function trialView (request, reply) {
 
             const endDate = processRules(ruleValues, Date.now());
 
+            if (request.query.requestType === 'ajaxRequest') {
```

Comment:
There are two questions there:
Do you need to? No, it can work mixed together.
Should it be changed? Yes, the change would bring it inline with our code quality policy.